.travis.yml

```diff
@@ -58,4 +58,4 @@ matrix:
     # - <<: *EXTENDED
     #   compiler: clang
     - os: osx
-      script: sh scripts/icl-homebrew-test
+      script: sh scripts/icl-homebrew-test.sh
```

Comment:
Actually, I meant to not simply _call_ the script from Travis, but to _get rid of_ the script and use the [Travis' way to install osx dependencies](https://docs.travis-ci.com/user/reference/osx) and finally build as usual.